src/index.js

```diff
@@ -1 +1,64 @@
 'use strict';
+/* eslint-disable no-unused-expressions */
+require('dotenv').config();
+
+const express = require('express');
+const cors = require('cors');
+const { WebSocketServer } = require('ws');
+const { authRoute } = require('./routes/auth.route');
+const { errorMiddleware } = require('./middlewares/errorMiddleware');
+const { userRoute } = require('./routes/users.route');
+const cookieParser = require('cookie-parser');
+const { roomsRoute } = require('./routes/rooms.route');
+const { messageRouter } = require('./routes/message.route');
+const { messageEmitter } = require('./controllers/messanger.controller');
```

Comment:
The import statement for `messageEmitter` is incorrect. It should be imported from `message.controller.js` instead of `messanger.controller.js`.